src/components/FileTabs/FileTabs.tsx

```diff
@@ -30,12 +30,13 @@ export type FileTabStatusType =
   | "warning"
   | "info";
 
-const TabsContainer = styled.div`
+const TabsContainer = styled.div<{ $count: number }>`
   display: flex;
   position: relative;
   overflow: auto;
   overscroll-behavior: none;
   scrollbar-width: 0;
+  max-width: ${({ $count }) => `${$count * 200}px`};
```

Comment:
how is the tab width set? what if we change it?

wouldn't something like `width: fit-content` or `display: inline-block` fix the issue?

Reply:
No, the issue is that we are not using the width 100% and accessing the value so the clamp is having difficulty deciding the width correctly
I have also experimented using min-width and max-width and that did not give the desired result